sql/hive/compatibility/src/test/scala/org/apache/spark/sql/hive/execution/HiveWindowFunctionQuerySuite.scala

```diff
@@ -472,7 +475,7 @@ class HiveWindowFunctionQuerySuite extends HiveComparisonTest with BeforeAndAfte
       |window w1 as (distribute by p_mfgr sort by p_mfgr, p_name
       |             rows between 2 preceding and 2 following)
     """.stripMargin, reset = false)
-
+  */
```

Comment:
I have added a statistical test case to the ```DataFrameWindowSuite```, this however doesn't match the level of thoroughness of the disabled tests. I have tried to add these tests verbatim to a separate test suite, but this becomes quite verbose and ugly (in style) and I rather not add this. I could expand the tests in the ```DataFrameWindowSuite```.